### ISSUE TITLE
[bert_run.py]145-149行验证集和测试集划分问题

### ISSUE THREAD

**Issue by sxphealer** [CCWUCMCTS/sentiment_classification_COVID-19#1]
train_inputs 和  valid_inputs 是用了相同的数据吗？

**Comment by CCWUCMCTS** [comment]
抱歉，才看到。
不是的，(train_idx, valid_idx)返回的是不同的索引，具体可以参见[这里](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.StratifiedKFold.html#sklearn.model_selection.StratifiedKFold)。

**Comment by CCWUCMCTS** [comment]
都是出自inputs，即已经标记的数据，但划分成两个部分，一部分是训练集，一部分是验证集。而最后需要提交识别结果作为得分依据的是测试集。

**Comment by sxphealer** [comment]
> 都是出自inputs，即已经标记的数据，但划分成两个部分，一部分是训练集，一部分是验证集。而最后需要提交识别结果作为得分依据的是测试集。

好的，谢谢楼主，还有一个问题，我跑这个代码，测试集正常，acc能到0.76。但是验证集acc loss mae 一直0，只替换了bert的中文模型。checkpoint我没转成h5，麻烦楼主了。

**Comment by CCWUCMCTS** [comment]
> > 都是出自inputs，即已经标记的数据，但划分成两个部分，一部分是训练集，一部分是验证集。而最后需要提交识别结果作为得分依据的是测试集。
> 
> 
> 
> 好的，谢谢楼主，还有一个问题，我跑这个代码，测试集正常，acc能到0.76。但是验证集acc loss mae 一直0，只替换了bert的中文模型。checkpoint我没转成h5，麻烦楼主了。

不好意思，这个我也不太清楚，当时是用小数据测试以后，直接把这个放到bitahub上跑的，没有碰到你说的问题。(・Д・)

**Comment by sxphealer** [comment]
好的 十分感谢&nbsp;

-------------------------------------------------------------------- 






姓名：石小平&nbsp;
学历：硕士研究生&nbsp;
学号：2018201916
学院：网络空间安全学院
专业：网络空间安全
qq:614951695
学校Email：shixiaoping@mail.nwpu.edu.cn
西北工业大学长安校区 &nbsp; 710129

&nbsp;


&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "CCWUCMCTS"<notifications@github.com&gt;; 
发送时间: 2020年11月30日(星期一) 晚上10:57
收件人: "CCWUCMCTS/sentiment_classification_COVID-19"<sentiment_classification_COVID-19@noreply.github.com&gt;; 
抄送: "Fugitive "<sxpcindy@foxmail.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [CCWUCMCTS/sentiment_classification_COVID-19] [bert_run.py]145-149行验证集和测试集划分问题 (#1)





   
都是出自inputs，即已经标记的数据，但划分成两个部分，一部分是训练集，一部分是验证集。而最后需要提交识别结果作为得分依据的是测试集。
  
好的，谢谢楼主，还有一个问题，我跑这个代码，测试集正常，acc能到0.76。但是验证集acc loss mae 一直0，只替换了bert的中文模型。checkpoint我没转成h5，麻烦楼主了。
  
不好意思，这个我也不太清楚，当时是用小数据测试以后，直接把这个放到bitahub上跑的，没有碰到你说的问题。(・Д・)
